### ISSUE TITLE
Some default scripts not marked as safe

### ISSUE THREAD

**Issue by djcater** [nmap/nmap#725]
```
$ nmap --script-help="default and not safe" 
```
(Output trimmed.)
```
Starting Nmap 7.40 ( https://nmap.org ) at 2017-03-04 00:28 GMT

dns-nsid
Categories: discovery default
https://nmap.org/nsedoc/scripts/dns-nsid.html

maxdb-info
Categories: default version
https://nmap.org/nsedoc/scripts/maxdb-info.html

sstp-discover
Categories: discovery default
https://nmap.org/nsedoc/scripts/sstp-discover.html

wdb-version
Categories: default version discovery vuln
https://nmap.org/nsedoc/scripts/wdb-version.html
```

It seems to me that the default category should only include safe scripts (within reason as there's always some risk with running a scan against a device). So either the above scripts should not be in the default category, or (more likely) they should be added to the safe category as well.

**Comment by Varunram** [comment]
Good catch there! I've made the changes in the PR. About the default category including only safe scripts, that's up for debate. Do refer https://nmap.org/book/nse-usage.html#nse-categories for formal definitions of both ```default``` and ```safe``` categories.

**Comment by dmiller-nmap** [comment]
Closed via #728
